rosidl_generator_py/resource/_msg.py.em

```diff
@@ -364,7 +370,7 @@ if isinstance(type_, AbstractNestedType):
         typename.pop()
         typename.append(self.__class__.__name__)
         args = []
-        for s, t in zip(self.__slots__, self.SLOT_TYPES):
+        for s, t in self.get_fields_and_field_types().items():
```

Comment:
This needs to be fixed to use `zip(self.get_fields_and_field_types().keys, self.SLOT_TYPES)`

Reply:
Oh you are right.

Reply:
Done.